### PR TITLE
Added ppc64le support to KCP image

### PR DESCRIPTION
**Pull request by mayurwaghmode**
## Summary
This PR adds PPC64Le support to the KCP image. It has used qemu user static for emulating the PPC64Le architecture.

## Related issue(s)

Fixes #

### CODE REVIEW

File: .github/workflows/kcp-image.yaml [hunk 1]
```diff
@@ -28,13 +28,17 @@ jobs:
...
+        uses: docker/setup-qemu-action@v1
+
       # Build and push a KCP image, tagged with latest and the commit SHA.
       - name: Build KCP Image
         id: build-image
         uses: redhat-actions/buildah-build@v2
         with:
           image: kcp
           tags: latest ${{ steps.vars.outputs.sha_short }} ${{ github.ref_name }}
+          archs: amd64, ppc64le
```
Comment (sttts) [inline .github/workflows/kcp-image.yaml:41]: can we add arm64 too while we are at it?

**Comment by ncdc** [comment]
FYI @stevekuznetsov - we'll eventually need to port this to prow

**Comment by sttts** [comment]
> we'll eventually need to port this to prow

Do we?

File: .github/workflows/kcp-image.yaml (hunk 1 above)
Comment (mayurwaghmode) [reply .github/workflows/kcp-image.yaml]: @sttts done 
Can u please review it again :)

**Pull request merged**